src/form/types/types.ts

```diff
@@ -491,6 +495,48 @@ export interface ISignatureFormField extends IFormFieldBase {
   )[]
 }
 
+export interface IHttpFormField extends IFormFieldBase {
```

Comment:
@tahmidrahman-dsi should we be adding these types into the common [countryconfig](https://github.com/opencrvs/opencrvs-countryconfig) repo? What do you think?

Reply:
Yes I think, so